spec/amp-actions-and-events.md

```diff
@@ -666,8 +666,8 @@ actions that apply to the whole document.
     <td>Tries to close the window if allowed, otherwise it navigates similar to <code>navigateTo</code> Action. Useful for use-cases where a "Back" button may need to close the window if it were opened in a new window from previous page or navigate if it wasn't opened.</td>
   </tr>
   <tr>
-    <td><code>goBack</code></td>
-    <td>Navigates back in history.</td>
+    <td><code>goBack(navigate=BOOLEAN)</code></td>
+    <td>Navigates back in history. `navigate` is optional, and if set to true, allows the action to navigate to an earlier page similar to [history.back](https://developer.mozilla.org/en-US/docs/Web/API/History/back).</td>
```

Comment:
I think this description is kinda more confusing now. The issue is the original description ("navigates back in history") is not actually true in all cases. 

Can we accurately describe both cases?